src/ioc/index.ts

```diff
@@ -1,12 +1,23 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
+
+const { api } = (window as any).__CONFIG__;
 
 export const createIoCContainer = () =>  {
   const ioc = new IoCContainer();
   // you can register some resources right now below...
+  ioc.register('apiConfig', api);
+  ioc.registerClass('logger', Logger);
+  ioc.registerClass('http', HTTP);
+  ioc.registerClass('users', Users);
 
   return ioc;
 };
+
+export const apiConfig = createIoCContainer().resolve('apiConfig');
+export const logger = createIoCContainer().resolve('logger');
+export const http = createIoCContainer().resolve('http');
+export const users = createIoCContainer().resolve('users');
```

Comment:
You should create a container only once and then use it.
It's better to resolve resources when and where you need them.
Tip: in this project, you need to resolve only `users` and `logger` services